### DISCUSSION TITLE
Pullrequest-backed changes

### DISCUSSION THREAD

**Discussion by vadi2** [Budibase/budibase#12375] [Help] [answered]
To help ensure quality, is there a way for changes to go through a pull request or a code review before deployment?

**Comment by melohagan** [comment, marked as answer]
Hey @vadi2 

Not at the moment, however there are a few features that make collaborative development a bit easier:

- https://docs.budibase.com/docs/audit-logs
- https://docs.budibase.com/docs/backups-1
- https://docs.budibase.com/docs/multi-dev-collaboration
